pkg/storj/object.go

```diff
@@ -26,7 +26,7 @@ var (
 // Bucket contains information about a specific bucket
 type Bucket struct {
 	Name                 string
-	Attribution          string
+	Attribution          string // TODO: this should be uuid.UUID
```

Comment:
these should be user agent formatted strings? not uuids

Reply:
Not according to everywhere else in the code base. For example, see https://github.com/storj/storj/blob/f9ed0dc1a83c0b7e6debef465ac846af56192885/satellite/attribution/db.go#L18-L24